src/QA/QA.Web/Program.cs

```diff
@@ -56,10 +56,17 @@ public static void Main(string[] args)
                 .UseNLog()
                 .ConfigureLogging(b => b.ConfigureRecruitLogging());
 
-        private static void CheckInfrastructure(IServiceProvider serviceProvider)
+        private static void CheckInfrastructure(IServiceProvider serviceProvider, NLog.ILogger logger)
         {
-            var collectionChecker = (MongoDbCollectionChecker)serviceProvider.GetService(typeof(MongoDbCollectionChecker));
-            collectionChecker.EnsureCollectionsExist();
+            try
+            {
+                var collectionChecker = (MongoDbCollectionChecker) serviceProvider.GetService(typeof(MongoDbCollectionChecker));
+                collectionChecker.EnsureCollectionsExist();
+            }
+            catch (Exception ex)
+            {
+                logger.Error(ex, "Error checking infrastructure");
```

Comment:
Would we not want the message to be a little more specific about collections missing.

Reply:
That gets logged out in the ex message